Update deploy-helm-charts.md (#1123)
harness/developer-hub: docs/continuous-delivery/cd-advanced/cd-helm-category/deploy-helm-charts.md

You can set up a Harness trigger to listen on the chart repo and execute the pipeline when a new chart version appears. For more information, go to [Trigger Pipelines on New Helm Chart](../../../platform/11_Triggers/trigger-pipelines-on-new-helm-chart.md).

## Option: Fetch Helm Chart Dependencies

> Harness can fetch Helm Chart Dependencies within Github using the `--dependency-update` command flag. Harness will be able to fetch dependent Helm Charts along with the main Helm Chart being targeted for deployment. The Dependencies will be resolved before Harness performs deployment of the primary Helm Chart that is configured with the service explicitly. 

Reviewer: ```suggestion
Harness can fetch Helm chart dependencies within Github using the `--dependency-update` command flag. Harness fetches dependent Helm charts along with the main Helm chart being targeted for the deployment. The dependencies will be resolved before Harness performs the deployment of the primary Helm chart configured with the service explicitly. 
```

> For More Details please review the [Helm Docs](https://helm.sh/docs/helm/helm_template/#helm)

Reviewer: ```suggestion
For more information, go to [Helm Docs](https://helm.sh/docs/helm/helm_template/#helm).
```

> To update Helm Chart dependencies, user's can configure:

Reviewer: ```suggestion
To update Helm chart dependencies:
```

> 1. If this is K8s Deployment then they can configure Template command with `--dependency-update`

Reviewer: ```suggestion
* For Kubernetes deployments, configure the template command with `--dependency-update`.
```

> 2. If this is Native K8s then they can add the same flag (`--depdency-update`) to `Install` and `Upgrade` commands

Reviewer: ```suggestion
* For Native Kubernetes deployments, add the command flag, `--depdency-update` to `Install` and `Upgrade` commands.
```

> **But in this case all the dependency repos should be available and accessible from delegate**

Reviewer: ```suggestion
:::info
All dependency repositories must be available and accessible from delegate.
:::
```

> Example below:

Reviewer: ```suggestion
Here is an example:
```

You can set up a Harness trigger to listen on the chart repo and execute the pipeline when a new chart version appears. For more information, go to [Trigger Pipelines on New Helm Chart](../../../platform/11_Triggers/trigger-pipelines-on-new-helm-chart.md).

> ## Option: Fetch Helm Chart Dependencies

Reviewer: ```suggestion
## Option: Fetch Helm chart dependencies
```

> Harness can fetch Helm chart dependencies within Github using the `--dependency-update` command flag. Harness fetches dependent Helm charts along with the main Helm chart being targeted for the deployment. These dependencies will be resolved before Harness performs the deployment of the primary Helm chart configured with the service explicitly. 

Reviewer: ```suggestion
Harness can fetch Helm chart dependencies within GitHub using the `--dependency-update` command flag. Harness fetches dependent Helm charts along with the main Helm chart being targeted for the deployment. These dependencies will be resolved before Harness performs the deployment of the primary Helm chart configured with the service explicitly. 
```
